app/routes/meetings/components/MeetingDetail.tsx

```diff
@@ -161,6 +162,9 @@ const MeetingDetails = () => {
   const statusMe = currentUserInvitation?.status;
   const actionGrant = meeting?.actionGrant;
   const canEdit = actionGrant?.includes('edit');
+  const pizzaActivated =
```

Comment:
Not sure how I feel about this variable name. I feel like it can be more descriptive

Reply:
Is it better now? I really open for suggestions